github/GithubObject.py

```diff
@@ -101,89 +144,93 @@ def __init__(self, requester, headers, attributes, completed):
         if self.CHECK_AFTER_INIT_FLAG:  # pragma no branch (Flag always set in tests)
             requester.check_me(self)
 
-    def _storeAndUseAttributes(self, headers, attributes):
+    def _storeAndUseAttributes(
+        self, headers: Dict[str, Union[str, int]], attributes: Any
+    ) -> None:
         # Make sure headers are assigned before calling _useAttributes
         # (Some derived classes will use headers in _useAttributes)
         self._headers = headers
         self._rawData = attributes
         self._useAttributes(attributes)
 
     @property
-    def raw_data(self):
+    def raw_data(self) -> Dict[str, Any]:
         """
         :type: dict
         """
         self._completeIfNeeded()
         return self._rawData
 
     @property
-    def raw_headers(self):
+    def raw_headers(self) -> Dict[str, Union[str, int]]:
         """
         :type: dict
         """
         self._completeIfNeeded()
         return self._headers
 
     @staticmethod
-    def _parentUrl(url):
+    def _parentUrl(url: str) -> str:
         return "/".join(url.split("/")[:-1])
 
     @staticmethod
-    def __makeSimpleAttribute(value, type):
+    def __makeSimpleAttribute(value: Any, type: Type[T]) -> Attribute[T]:
         if value is None or isinstance(value, type):
-            return _ValuedAttribute(value)
+            return _ValuedAttribute(value)  # type: ignore
         else:
-            return _BadAttribute(value, type)
+            return _BadAttribute(value, type)  # type: ignore
 
     @staticmethod
-    def __makeSimpleListAttribute(value, type):
+    def __makeSimpleListAttribute(value: list, type: Type[T]) -> Attribute[T]:
         if isinstance(value, list) and all(
             isinstance(element, type) for element in value
         ):
-            return _ValuedAttribute(value)
+            return _ValuedAttribute(value)  # type: ignore
         else:
-            return _BadAttribute(value, [type])
+            return _BadAttribute(value, [type])  # type: ignore
 
     @staticmethod
-    def __makeTransformedAttribute(value, type, transform):
+    def __makeTransformedAttribute(
+        value: Any, type: Type[T], transform: Callable[..., Any]
```

Comment:
Shouldn't `value` be of type `T` and callable be `Callable[[T], Any]`?

Reply:
This method returns `_BadAttribute` if `value` is not of type `T`, so `value` can be of any type.